frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-runs.feature

```diff
@@ -295,3 +295,51 @@ Feature: Pipeline Runs
               And user clicks on Start
              Then user will be redirected to Pipeline Run Details page
               And user will see PVC Workspace "workspace-ex" mentioned in the VolumeClaimTemplate Resources section of Pipeline Run Details page
+
+
+        @regression, @manual
+        Scenario: Pipeline Run details page with passed pipeline run having finally task
+            Given pipeline with finally task having a passed pipeline run is present
+              And user is on Pipeline Run details page of passed pipeline run
+             When user scrolls over each of the finally task
+             Then user can see finally tasks arranged in parallel with each task as passed
+              And user can see tooltip with task name and Finally task mentioned while scrolling
+
+
+        @regression, @manual
+        Scenario: Pipeline Run details page with failed pipeline run having finally task
+            Given user has pipeline "pipeline-finally-failed" with three tasks in series and a finally task
+              And user has failed pipeline run for pipeline "pipeline-finally-failed" with third task as failure
+              And user is on Pipeline Run details page of failed pipeline run
+             When user scrolls over each of the finally task
+             Then user can see finally tasks arranged in parallel with each task as passed
+              And user can see failed task before finally task
+              And user can see tooltip with task name and Finally task mentioned while scrolling
+
+
+        @regression @to-do
+        Scenario: Pipeline Run results on Pipeline Run details page for passed pipeline run
+            Given user is on Pipeline Run details page of passed pipeline run
+             When user scrolls to the Pipeline Run results section
+             Then user can see Name and Value column under Pipeline Run results
+
+
+        @regression @to-do
+        Scenario: Pipeline Run results on Pipeline Run details page for failed pipeline run
+            Given user is on Pipeline Run details page of failed pipeline run
+             When user scrolls to the Pipeline Run results section
+             Then user can see message "No Pipeline Run results available due to failure."
+
+
+        @regression, @manual
+        Scenario: Pipeline Run details page with passed pipeline run having Conditional task
+            Given user is on Pipeline Run details page of passed pipeline run having a Conditional task
+             When user scrolls over the diamond before the Conditional task
+             Then user can see tooltip with "When expression was met" mentioned on it
+
+
```

Comment:
Please add the scenario for pipeline Details page, having the diamond shape (no color applied) for conditional and on hovering over it "When expression" tooltip appears.

Reply:
I've added that as part of the story https://issues.redhat.com/browse/ODC-5420 in pr https://github.com/openshift/console/pull/8501